osgp/platform/osgp-adapter-ws-admin/src/main/java/org/opensmartgridplatform/adapter/ws/admin/application/services/DeviceManagementService.java

```diff
@@ -653,6 +655,44 @@ public void updateDeviceProtocol(
         deviceIdentification);
   }
 
+  public Device updateCommunicationNetworkInformation(
+      final String deviceIdentification,
+      final String ipAddress,
+      final Integer btsId,
+      final Integer cellId)
+      throws FunctionalException {
+
```

Comment:
OK, I missed that. Actually I notice that as a SMHE developer I miss a lot of GXF functional knowledge. Perhaps in the future it would be better that the GXF team implements the service.

Reply:
I added the proper device function.